examples/webgl_loader_mmd.html

```diff
@@ -160,23 +172,76 @@
 
 				if( mesh ) {
 
-					mixer.update( clock.getDelta() );
-
-					/* Note: ikSolver.resetLinks() is workaround for r74dev.
-					 *       THREE.PropertyMixer.apply() seems to have buffer cache
-					 *       when mixer.update() is called.
-					 *       ikSolver.update() change bone position/quaternion
-					 *       without mixer.update() then buffer cache will be inconsistent.
-					 *       ikSolver.resetLinks() explicitly resets IK link bones quaternion and
-					 *       tries to avoid issues caused by this inconsistent.
-					 */
-					ikSolver.resetLinks();
+					var delta = clock.getDelta();
+					mixer.update( delta );
+
+					// see below for what this's for
+					backupBones( mesh );
+
 					ikSolver.update();
+					physics.update( delta );
 
 				}
 
 				renderer.render( scene, camera );
 
+				if ( mesh ) {
+
+					// see below for what this's for
+					restoreBones( mesh );
+
+				}
+
+			}
+
+			/*
+			 * Note: These following three functions are workaround for r74dev.
+			 *       THREE.PropertyMixer.apply() seems to save values into buffer cache
```

Comment:
> I'm not quite sure the detail of Three.js animation system inside so can't say yet if it's operatable on PropertyMixers.
> [...]
> I think there'd better to have a way to add user-plugin for mixer because there're some animation techniques which dynamically address the bones like these IK and Physics.

Plugins would have to work on `PropertyMixer`s. Those are `PropertyBinding`s plus an array that holds the buffers where the mixing takes place. `PropertyBinding` implements the actual access to properties by path name.

`AnimationMixer` does all the memory management, so the plugin will have to advertise the path names of the properties it wants to access and get the corresponding `PropertyMixer` instances for it, somewhow. Similar to what `AnimationMixer` does with its `._Action`s.

Plugins would update after accumulation and before applying the state. 

I guess I wrote most of the code you'd have to hook into and I'm here to help if you want to give it a shot.